marklogic-junit5/build.gradle

```diff
@@ -14,26 +14,23 @@ repositories {
 }
 
 dependencies {
-  compile project(":marklogic-unit-test-client")
-	compile "com.marklogic:ml-javaclient-util:3.11.0"
-	compile "jaxen:jaxen:1.1.6"
-	compile "org.junit.jupiter:junit-jupiter-api:5.4.1"
-	compile "org.junit.jupiter:junit-jupiter-params:5.4.1"
-	compile "org.springframework:spring-context:5.0.8.RELEASE"
-	compile "org.springframework:spring-test:5.0.8.RELEASE"
-
-	// Support for DHF is provided, but a client must specify their own version of the DHF library to use
-	compileOnly ("com.marklogic:marklogic-data-hub:4.2.2") {
-		exclude module: "ml-javaclient-util"
-	}
+  api project(":marklogic-unit-test-client")
+	api "com.marklogic:ml-javaclient-util:4.1.1"
+	api "jaxen:jaxen:1.1.6"
+	api "org.junit.jupiter:junit-jupiter-api:5.4.1"
```

Comment:
I recommend bumping the JUnit dependencies to the latest versions too - I don't think it will have any impact on the code, but run the tests just to be sure.